src/utilities/getBadgeData.js

```diff
@@ -7,17 +7,30 @@ const routeList = {
   supplierInvoices: 'SupplierInvoice',
   stocktakes: 'Stocktake',
   customerInvoices: 'CustomerInvoice',
+  vaccines: 'TemperatureBreach',
 };
 
 const getBadgeData = routeName => {
   const dataType = routeName in routeList ? routeList[routeName] : '';
 
-  return [
-    {
-      count: dataType ? UIDatabase.objects(dataType).filtered('status != "finalised"').length : 0,
-      title: `${generalStrings.unfinalised} ${generalStrings[routeName]}`,
-    },
-  ];
+  switch (dataType) {
+    case 'TemperatureBreach':
+      return [
+        {
+          count: UIDatabase.objects(dataType).filtered('acknowledged == false').length,
```

Comment:
won't this give the number of breaches? The issue was asking for the number of sensors with a breach

Reply:
Yeah, this will give the total number of breaches that have not yet been acknowledged, with the intention to follow it up with a way to acknowledge them. 

Although maybe if we need something to demo ASAP (like Monday) then we can update this one to just number of sensors with a breach, I think I've confused myself with which option we would be doing now 🤣

Reply:
The text on the issue is "Badge should display the number of sensors with current breaches in progress"
( I thought it originally asked for in progress and finished breaches.. but 🤷‍♂️  )
is this possible?
`count: UIDatabase.objects(dataType).filtered('acknowledged == false').distinct('sensor').length`

Reply:
no, it isn't. `distinct` is on collection only sorry.

Reply:
Thanks, yeah I had tried that and it didn't like that 🤣 . I tried another way, clear enough hopefully?
a20b391
dac5b6c